### ISSUE TITLE
`quail -u` fails to request a web page in Ubuntu 14.04; works fine in 12.04

### ISSUE THREAD

**Issue by jessebeach** [quailjs/quail#318]

**Comment by bartfeenstra** [comment]
```
vagrant@vagrant-ubuntu-trusty-64:/opt/quail$ sudo ./bin/quail -u http://drupal.org -R wcag2
Fontconfig warning: ignoring UTF-8: not a valid region tag
["Requested (GET)","http://drupal.org/"]
["Resource timeout",408,"Network timeout on resource.","http://drupal.org/"]
[
  "Resource error",
  "Error code: 5",
  "Operation canceled",
  "http://drupal.org/"
]
["Received",null,"http://drupal.org/"]
^Cvagrant@vagrant-ubuntu-trusty-64:/opt/quail$
```

This happens on Ubuntu 14.04, but not on 12.04 or Mac OS 10.10. The only things I know is that this does not seem to be a networking or SSL problem, as both Ubuntu installations were located in a VM on Mac OS 10.10. I also called PhantomJS manually using the same command as the WCAG2 test runner uses, but with SSL errors ignored. I tried a number of different hosts within my own country as well as on different continents, but the requests consistently failed for all hosts on Ubuntu 12.04 only.

**Comment by bartfeenstra** [comment]
To reproduce these failures:
- Create a bare VM using the `ubuntu/trusty64` base box.
- `vagrant ssh`
- Clone Quail.
- `npm install` and `grunt build`
- Run a Quail test.
